buildstockbatch/workflow_generator/commercial.py

```diff
@@ -146,6 +152,7 @@ def create_osw(self, sim_id, building_id, upgrade_idx):
             }
         ])
 
+        # FIXME: Insert the reporting measures somewhere around here
```

Comment:
@asparke2 I added the `reporting_measures` argument and validation to the commercial workflow generator, but I stopped short of implementing it because I didn't know where you wanted the measures added to your workflow. 

Here's how I did it for the residential workflow. Yours will probably be very similar. 

https://github.com/NREL/buildstockbatch/blob/dc2cbe61a9ca1b40d8c35b448c3f4753cbfc6dd0/buildstockbatch/workflow_generator/residential.py#L336-L341